<?xml version="1.0" encoding="UTF-8"?>
<ui version="4.0">
 <class>Form</class>
 <widget class="QWidget" name="Form">
  <property name="geometry">
   <rect>
    <x>0</x>
    <y>0</y>
    <width>1210</width>
    <height>943</height>
   </rect>
  </property>
  <property name="windowTitle">
   <string>Form</string>
  </property>
  <widget class="QPushButton" name="pushButton">
   <property name="geometry">
    <rect>
     <x>10</x>
     <y>40</y>
     <width>221</width>
     <height>41</height>
    </rect>
   </property>
   <property name="font">
    <font>
     <pointsize>13</pointsize>
     <weight>75</weight>
     <bold>true</bold>
    </font>
   </property>
   <property name="styleSheet">
    <string notr="true">background-color: rgb(255, 7, 73);</string>
   </property>
   <property name="text">
    <string>Load Image</string>
   </property>
  </widget>
  <widget class="QGraphicsView" name="graphicsView_2">
   <property name="geometry">
    <rect>
     <x>10</x>
     <y>300</y>
     <width>241</width>
     <height>61</height>
    </rect>
   </property>
   <property name="styleSheet">
    <string notr="true">background-color: rgb(156, 153, 255);</string>
   </property>
  </widget>
  <widget class="QLabel" name="label_8">
   <property name="geometry">
    <rect>
     <x>20</x>
     <y>300</y>
     <width>161</width>
     <height>41</height>
    </rect>
   </property>
   <property name="font">
    <font>
     <pointsize>12</pointsize>
     <weight>75</weight>
     <bold>true</bold>
    </font>
   </property>
   <property name="text">
    <string>Morophology</string>
   </property>
  </widget>
  <widget class="QGraphicsView" name="graphicsView_3">
   <property name="geometry">
    <rect>
     <x>310</x>
     <y>70</y>
     <width>791</width>
     <height>701</height>
    </rect>
   </property>
   <property name="styleSheet">
    <string notr="true">background-color: qlineargradient(spread:pad, x1:0, y1:0, x2:1, y2:0, stop:0 rgba(0, 0, 0, 255), stop:1 rgba(255, 255, 255, 255));</string>
   </property>
  </widget>
  <widget class="QGraphicsView" name="graphicsView_4">
   <property name="geometry">
    <rect>
     <x>10</x>
     <y>90</y>
     <width>241</width>
     <height>181</height>
    </rect>
   </property>
   <property name="styleSheet">
    <string notr="true">background-color: rgb(144, 161, 255);</string>
   </property>
  </widget>
  <widget class="QLabel" name="label_10">
   <property name="geometry">
    <rect>
     <x>20</x>
     <y>620</y>
     <width>151</width>
     <height>31</height>
    </rect>
   </property>
   <property name="font">
    <font>
     <pointsize>10</pointsize>
     <weight>75</weight>
     <bold>true</bold>
    </font>
   </property>
   <property name="text">
    <string>Segmentation</string>
   </property>
  </widget>
  <widget class="QLabel" name="label_12">
   <property name="geometry">
    <rect>
     <x>20</x>
     <y>700</y>
     <width>171</width>
     <height>21</height>
    </rect>
   </property>
   <property name="font">
    <font>
     <pointsize>10</pointsize>
     <weight>75</weight>
     <bold>true</bold>
    </font>
   </property>
   <property name="text">
    <string/>
   </property>
  </widget>
  <widget class="QGraphicsView" name="graphicsView_5">
   <property name="geometry">
    <rect>
     <x>10</x>
     <y>380</y>
     <width>241</width>
     <height>221</height>
    </rect>
   </property>
   <property name="styleSheet">
    <string notr="true">background-color: rgb(167, 167, 255);</string>
   </property>
  </widget>
  <widget class="QLabel" name="label_15">
   <property name="geometry">
    <rect>
     <x>20</x>
     <y>390</y>
     <width>181</width>
     <height>31</height>
    </rect>
   </property>
   <property name="font">
    <font>
     <pointsize>11</pointsize>
     <weight>75</weight>
     <bold>true</bold>
    </font>
   </property>
   <property name="text">
    <string>Smoothing -Filter</string>
   </property>
  </widget>
  <widget class="QLabel" name="label_16">
   <property name="geometry">
    <rect>
     <x>10</x>
     <y>510</y>
     <width>221</width>
     <height>21</height>
    </rect>
   </property>
   <property name="font">
    <font>
     <pointsize>10</pointsize>
     <weight>75</weight>
     <bold>true</bold>
    </font>
   </property>
   <property name="text">
    <string>Translation and Rotation</string>
   </property>
  </widget>
  <widget class="QSlider" name="horizontalSlider">
   <property name="geometry">
    <rect>
     <x>30</x>
     <y>430</y>
     <width>160</width>
     <height>22</height>
    </rect>
   </property>
   <property name="orientation">
    <enum>Qt::Horizontal</enum>
   </property>
  </widget>
  <widget class="QSlider" name="horizontalSlider_2">
   <property name="geometry">
    <rect>
     <x>30</x>
     <y>460</y>
     <width>161</width>
     <height>22</height>
    </rect>
   </property>
   <property name="orientation">
    <enum>Qt::Horizontal</enum>
   </property>
  </widget>
  <widget class="QSlider" name="horizontalSlider_3">
   <property name="geometry">
    <rect>
     <x>30</x>
     <y>490</y>
     <width>171</width>
     <height>22</height>
    </rect>
   </property>
   <property name="orientation">
    <enum>Qt::Horizontal</enum>
   </property>
  </widget>
  <widget class="QSlider" name="horizontalSlider_4">
   <property name="geometry">
    <rect>
     <x>30</x>
     <y>540</y>
     <width>171</width>
     <height>22</height>
    </rect>
   </property>
   <property name="orientation">
    <enum>Qt::Horizontal</enum>
   </property>
  </widget>
  <widget class="QPushButton" name="pushButton_2">
   <property name="geometry">
    <rect>
     <x>20</x>
     <y>100</y>
     <width>201</width>
     <height>28</height>
    </rect>
   </property>
   <property name="text">
    <string>perwit</string>
   </property>
  </widget>
  <widget class="QPushButton" name="pushButton_3">
   <property name="geometry">
    <rect>
     <x>20</x>
     <y>140</y>
     <width>201</width>
     <height>28</height>
    </rect>
   </property>
   <property name="text">
    <string>Robert</string>
   </property>
  </widget>
  <widget class="QPushButton" name="pushButton_4">
   <property name="geometry">
    <rect>
     <x>20</x>
     <y>180</y>
     <width>201</width>
     <height>28</height>
    </rect>
   </property>
   <property name="text">
    <string>Canny</string>
   </property>
  </widget>
  <widget class="QPushButton" name="pushButton_5">
   <property name="geometry">
    <rect>
     <x>20</x>
     <y>220</y>
     <width>201</width>
     <height>28</height>
    </rect>
   </property>
   <property name="text">
    <string>LOG</string>
   </property>
  </widget>
  <widget class="QPushButton" name="pushButton_7">
   <property name="geometry">
    <rect>
     <x>30</x>
     <y>690</y>
     <width>181</width>
     <height>28</height>
    </rect>
   </property>
   <property name="text">
    <string>Manual Seg</string>
   </property>
  </widget>
  <widget class="QPushButton" name="pushButton_8">
   <property name="geometry">
    <rect>
     <x>30</x>
     <y>650</y>
     <width>181</width>
     <height>28</height>
    </rect>
   </property>
   <property name="text">
    <string>Histogram Peak Seg</string>
   </property>
  </widget>
  <widget class="QPushButton" name="pushButton_9">
   <property name="geometry">
    <rect>
     <x>30</x>
     <y>730</y>
     <width>181</width>
     <height>28</height>
    </rect>
   </property>
   <property name="text">
    <string>Adaptive Histogram</string>
   </property>
  </widget>
  <widget class="QPushButton" name="pushButton_10">
   <property name="geometry">
    <rect>
     <x>30</x>
     <y>770</y>
     <width>181</width>
     <height>28</height>
    </rect>
   </property>
   <property name="text">
    <string>Otsu</string>
   </property>
  </widget>
  <widget class="QGraphicsView" name="graphicsView_6">
   <property name="geometry">
    <rect>
     <x>10</x>
     <y>620</y>
     <width>241</width>
     <height>211</height>
    </rect>
   </property>
   <property name="styleSheet">
    <string notr="true">background-color: rgb(144, 161, 255);</string>
   </property>
  </widget>
  <widget class="QPushButton" name="pushButton_11">
   <property name="geometry">
    <rect>
     <x>640</x>
     <y>810</y>
     <width>93</width>
     <height>28</height>
    </rect>
   </property>
   <property name="styleSheet">
    <string notr="true">background-color: rgb(85, 0, 255);</string>
   </property>
   <property name="text">
    <string>reset</string>
   </property>
  </widget>
  <widget class="QSlider" name="horizontalSlider_5">
   <property name="geometry">
    <rect>
     <x>30</x>
     <y>570</y>
     <width>171</width>
     <height>22</height>
    </rect>
   </property>
   <property name="orientation">
    <enum>Qt::Horizontal</enum>
   </property>
  </widget>
  <zorder>graphicsView_6</zorder>
  <zorder>pushButton</zorder>
  <zorder>graphicsView_2</zorder>
  <zorder>label_8</zorder>
  <zorder>graphicsView_3</zorder>
  <zorder>graphicsView_4</zorder>
  <zorder>label_10</zorder>
  <zorder>label_12</zorder>
  <zorder>graphicsView_5</zorder>
  <zorder>label_15</zorder>
  <zorder>label_16</zorder>
  <zorder>horizontalSlider</zorder>
  <zorder>horizontalSlider_2</zorder>
  <zorder>horizontalSlider_3</zorder>
  <zorder>horizontalSlider_4</zorder>
  <zorder>pushButton_2</zorder>
  <zorder>pushButton_3</zorder>
  <zorder>pushButton_4</zorder>
  <zorder>pushButton_5</zorder>
  <zorder>pushButton_7</zorder>
  <zorder>pushButton_8</zorder>
  <zorder>pushButton_9</zorder>
  <zorder>pushButton_10</zorder>
  <zorder>pushButton_11</zorder>
  <zorder>horizontalSlider_5</zorder>
 </widget>
 <resources/>
 <connections/>
</ui>
